sapphire/transformations/celestial.py

```diff
@@ -240,3 +264,91 @@ def galactic_to_equatorial(latitude, longitude, epoch='J2000'):
     dec = arcsin(cosb * cospoledec * sinlpos + sinb * sinpoledec)
 
     return ra, dec
+
+try:
+    # This try-except structure has been implemented to accommodate those without astropy.
+    import astropy.units as u
+    import datetime
+    import numpy as np
+
+    from astropy.coordinates import AltAz
+    from astropy.coordinates import EarthLocation
+    from astropy.coordinates import SkyCoord
```

Comment:
Combine multiple imports from the same module:
`from astropy.coordinates import AltAz, EarthLocation, SkyCoord`